opentech/apply/funds/templates/funds/tables/table.html

```diff
@@ -1,7 +1,8 @@
 {% extends 'django_tables2/table.html' %}
-{% load django_tables2 table_tags review_tags %}
+{% load adv_cache django_tables2 table_tags review_tags %}
 
 {% block table.tbody.row %}
+    {% cache 3600 submission_table_row row.record.pk row.record.status row.record.last_update %}
```

Comment:
What do you think of this solution?

This will cache each table row that contains a single submission. Need to test to get the correct cache tag but this is a good start I believe.

Would like to use ADV_CACHE_TIMEOUT here instead of hardcoded "3600". Can you help with that? Tried using context but did not get it to work.